Classes/Finishers/EmailFinisher.php

```diff
@@ -217,7 +217,7 @@ protected function addAttachments(SwiftMailerMessage $mail)
         if (is_array($attachmentConfigurations)) {
             foreach ($attachmentConfigurations as $attachmentConfiguration) {
                 if (isset($attachmentConfiguration['resource'])) {
-                    $mail->attach(\Swift_Attachment::fromPath($attachmentConfiguration['resource']));
+                    $mail->attach(new \Swift_Attachment($attachmentConfiguration['resource']));
```

Comment:
`fromPath` isn't removed, so this will behave differently (`$attachmentConfiguration['resource']` is a file path)